exams/pearson_test.py

```diff
@@ -99,16 +120,11 @@ def test_write_cdd_file(self):  # pylint: disable=no-self-use
 
         with mute_signals(post_save):
             exam_profiles = [ExamProfileFactory.create(**kwargs)]
+            exam_profiles[0].profile.updated_on = datetime(2016, 5, 15, 15, 2, 55)
```

Comment:
then please move  `datetime(2016, 5, 15, 15, 2, 55)` to a variable or just use `.now(tz=pytz.UTC)`